mwviews/api/pageviews.py

```diff
@@ -118,17 +117,21 @@ def article_views(
 
         urls = [
             '/'.join([
-                # granularity is faked on the client side, the API
-                # currently only supports daily
-                endpoints['article'], project, access, agent, a, 'daily',
+                endpoints['article'], project, access, agent, a, granularity,
                 format_date(startDate), format_date(endDate),
             ])
             for a in articlesSafe
         ]
 
         outputDays = timestamps_between(startDate, endDate, timedelta(days=1))
+        if granularity == 'monthly':
```

Comment:
Here you can just do:

    outputMonths = list(set([month_from_day(day) for day in outputDays]))

Reply:
Right, my bad... thanks!